svgdigitizer/entrypoint.py

```diff
@@ -599,6 +599,10 @@ def paginate(onlypng, template, pdf, outdir):
         >>> with TemporaryData("**/mustermann_2021_svgdigitizer_1.pdf") as directory:
         ...     invoke(cli, "paginate", os.path.join(directory, "mustermann_2021_svgdigitizer_1.pdf"))
 
+    TESTS::
+
+        >>> from svgdigitizer.test.cli import invoke, TemporaryData
+
```

Comment:
Considering the trouble with creating the documentation above, I think it would make sense to add some tests here